autopush/db.py

```diff
@@ -537,42 +537,59 @@ def __init__(self, tablename, boto_resource=None,
     def table_status(self):
         return self.table.table_status
 
+    @track_provisioned
+    def touch_record(self, uaid, ttl=None):
```

Comment:
should we kill this, then? it's now unused

Reply:
yep